<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="CDC2-L-025418W" ignored="0" name="Default test" tests="2" failures="0" timestamp="29 Dec 2020 22:57:08 GMT" time="0.264" errors="0">
  <testcase name="@BeforeTest setUp" time="0.199" classname="AdvancedConcepts.ExtentReport">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at AdvancedConcepts.ExtentReport.setUp(ExtentReport.java:39)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.beforeRun(TestRunner.java:529)
at org.testng.TestRunner.run(TestRunner.java:497)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- @BeforeTest setUp -->
  <testcase name="@AfterMethod chekResult" time="0.0" classname="AdvancedConcepts.ExtentReport">
    <skipped/>
  </testcase> <!-- @AfterMethod chekResult -->
  <testcase name="@AfterMethod chekResult" time="0.0" classname="AdvancedConcepts.ExtentReport">
    <skipped/>
  </testcase> <!-- @AfterMethod chekResult -->
  <testcase name="@AfterTest tearDown" time="0.0" classname="AdvancedConcepts.ExtentReport">
    <skipped/>
  </testcase> <!-- @AfterTest tearDown -->
  <testcase name="OpenBrowser" time="0.0" classname="AdvancedConcepts.ExtentReport">
    <skipped/>
  </testcase> <!-- OpenBrowser -->
  <testcase name="VerifyTitle" time="0.0" classname="AdvancedConcepts.ExtentReport">
    <skipped/>
  </testcase> <!-- VerifyTitle -->
</testsuite> <!-- Default test -->
